torch/package/package_exporter.py

```diff
@@ -419,6 +431,12 @@ def save_mock_module(self, module_name: str):
         is_package = hasattr(self._import_module(module_name), '__path__')
         self.save_source_string(module_name, _MOCK_IMPL, is_package, dependencies=False)
 
+    def reject_denied_module(self, module_name: str):
```

Comment:
nit: prefix with `_` as this is a private method. The equivalents for mock and deny have a valid reason to be public (users may want to explicitly extern/mock a specific module) but that's not really the case here.